tasks/docs.rb

```diff
@@ -153,19 +153,41 @@ def friendly_default_value(property)
       property["default"].to_s
     end
 
+    #
+    # Build the actions section of the resource yaml
+    #
+    # @return [Hash]
+    #
+    def action_list(actions)
+      list = {}
+      actions.sort.each do |action|
+        # skip it so we can make it the last value later
+        next if action == "nothing"
+
+        list[action.to_sym] = { "markdown" => nil }
+      end
+
+      # add the special case for nothing
+      list[:nothing] = { "shortcode" => "resources_common_actions_nothing.md" }
+
+      list
+    end
+
     # TODO:
     # - what to do about "lazy default" for default?
```

Comment:
```suggestion
```

Extremely nitpick, but I added this comment and then I think you addressed that a day or two later?

Reply:
the lazy bit is still there and I keep seeing this and wondering what the right solution is? Maybe some warnings in the script or something.

Reply:
Ahh ok.